packages/react-ui-core/src/Map/index.js

```diff
@@ -0,0 +1,2 @@
+export { default } from './Map'
+export { default as Mapbox } from './Map'
```

Comment:
The exported name should match the component, e.g. `export { default as Map } from './Map'`